WORKSPACE.in

```diff
@@ -203,3 +203,10 @@ new_local_repository(
     build_file = "third_party/adv_plat.BUILD",
     path = "/usr/local/apollo/adv_plat"
 )
+#libprotobuf-mutator
+new_git_repository(
+    name = "libprotobuf_mutator",
+    build_file = "third_party/libprotobuf_mutator.BUILD",
+    commit = "91a872998bc204e9484dccdbeeafc608c1ee8062",
+    remote = "https://github.com/jiayunhan/libprotobuf-mutator",
```

Comment:
Better to refer an official repo instead of personal.

Reply:
I forked this repo from Google Github with some modifications, where do you suggest to put the repo? Is it possible to put it together with Apollo, under ApolloAuto. I will fix other issues. Thanks.

Reply:
Never mind, I will put it under our  Baidu X-Lab Github organization. Thanks.